lib/bokken/documents.ex

```diff
@@ -138,4 +154,10 @@ defmodule Bokken.Documents do
   def change_file(%File{} = file, attrs \\ %{}) do
     File.changeset(file, attrs)
   end
+
+  def verify_total_size(file, user_id) do
+    user = Accounts.get_user!(user_id)
+
+    user.total_file_size + File.file_size(file)
+  end
```

Comment:
This name is misleading. It should be `get_new_total_size` or something like that.